packages/hint-create-element-svg/src/meta.ts

```diff
@@ -0,0 +1,15 @@
+import { Category } from 'hint/dist/src/lib/enums/category';
+import { HintScope } from 'hint/dist/src/lib/enums/hint-scope';
+import { HintMetadata } from 'hint/dist/src/lib/types';
+
+const meta: HintMetadata = {
+    docs: {
+        category: Category.pitfalls,
+        description: `Infrom users that they need to use createElementNS to create SVG elements instead of createElement`
```

Comment:
We support a "friendly name" in the metadata now, usually matching the `README` header (another place `create-hint` could use an update). Also fix `Infrom` to be `Inform` in the description.
```suggestion
        description: `Inform users that they need to use createElementNS to create SVG elements instead of createElement`,
        name: 'No `createElement` with SVG'
```